exceptions.py

```diff
@@ -0,0 +1,11 @@
+""" Exceptions used in configuration script """
```

Comment:
Can you actually just throw this in `comanage_utils.py`? Since we're installing these files directly via puppet (i.e., it's not packaged), it's more overhead to manage multiple disparate files.